x-pack/plugins/security_solution/server/endpoint/routes/resolver/utils/fetch.ts

```diff
@@ -138,6 +150,29 @@ export class Fetcher {
     );
   }
 
+  private async doAlerts(limit: number, after?: string) {
```

Comment:
out of curiosity, why not inline this logic in the `alerts` method above?

Reply:
Yeah good call. I'll refactor it in a follow up PR when I add the `.siem` index info 👍